changelog

```diff
@@ -1,3 +1,136 @@
+Version 0.10.5
+  * Big changes:
+    * Mining Additions (new attributes, new outfits, new ships, new mechanics, etc... will expand upon this before release).
+    * CCOR/Pirate Additions (will expand upon this before release).
```

Comment:
```suggestion
    * Added a small pirate region to the south, home to the remnants of a collapsed government: the Confederated Councils of the Outer Rim. Includes the addition of various new pirate ships and weapons. This is the first of a number of planned expansions to pirate content and gameplay, with the goal of making it so that playing as a pirate is a more viable option. Stay tuned for more in future updates. (@bene-dictator, @Ebreus, @Galaucus, @Quantumshark, @Saugia)
```

Reply:
Don't forget Becca, who modelled the Javelin Turret.